frontend/src/components/NavBar.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import {
+  Button,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+  Nav,
+  Navbar,
+  NavbarBrand,
+  NavItem,
+  UncontrolledDropdown
+} from 'reactstrap'
+import LanguageSelector from './LanguageSelector'
+import { withRouter } from 'react-router-dom'
+import k_logo from '../media/greenK.png'
+import kiva_logo from '../media/kivaPlainLogo.png'
+import info_image from '../media/gray_info.png'
+import sandwich_image from '../media/sandwich.png'
+import '../styles/navbar.css'
+
+const mapStateToProps = state => ({
+  isPM: state.user.isPM
+})
+
+class NavBar extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      isLoginPage: null
+    }
+  }
+
+  componentDidMount() {
+    if (this.props.location.pathname === '/dashboard') {
+      this.setState({ isLoginPage: false })
```

Comment:
I would make the boolean expression here more general - every page has the same navbar except the login page, so i think a better condition would be this.path.location.pathname != '/'

Reply:
Since the login page is the home page, '/'